exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,9 @@
 // Declare your function here
 
+function createLongGreeting(name , age){
+ return `Hello my name is ${name} and I am ${age} years old`
```

Comment:
For example line 4 here, so simple. Mine looked a lot more complicated than that to get the same result. I will try to remember how you did this and use the same method in future. Thanks!

Reply:
You are welcome Alex!